android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -141,6 +147,42 @@ public boolean onTouchEvent(MotionEvent event) {
 		setNativeView(tv);
 
 	}
+	
+	/**
+	 * Method used to decrease the fontsize of the text to fit the width
+	 * fontsize should be >= than the property minimumFontSize
```

Comment:
Also, it looks like the other comment before was removed so:
+ refactor parameter name to `view` or change `@param view` to `@param v`